repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml

```diff
@@ -356,6 +358,9 @@
 
                 <!-- target-less transition to publish the document when no event payload parameter targetDate is provided -->
                 <transition event="publish" cond="!_event.data?.targetDate">
+                    <!-- Trigger publication of external files -->
+                    <externalstorage:externalFilePublish unpublishedVariant="unpublished"/>
```

Comment:
If the variant you need is always unpublished you don't need to specify it as a variable. Just access it directly from the document handle.